ios/ReactNativeVolumeController/ReactNativeVolumeController.m

```diff
@@ -40,6 +40,10 @@ - (void)stopObserving {
     hasListeners = NO;
 }
 
+- (dispatch_queue_t)methodQueue {
+  return dispatch_get_main_queue();
+}
+
 - (void)initAudioSessionObserver{
     audioSession = [AVAudioSession sharedInstance];
     [audioSession setActive:YES error:nil];
```

Comment:
Since you're only observing changes, you should care about the audiosession status at this point. Another library may have its own reasons for deactivating it for the moment.